src/operationsClient.ts

```diff
@@ -208,7 +220,7 @@ export class OperationsClient {
       protos.google.longrunning.GetOperationRequest,
       {} | null | undefined
     >
-  ): CancellablePromise<ResultTuple> {
+  ): Promise<protos.google.longrunning.Operation> {
```

Comment:
The fact that you changed the return value type and no tests required changes basically means that this is not covered by any tests :) Can we maybe have a small unit test that would check the operations client types?

Reply:
Unfortunately, we have no TypeScript integration test here in gax; showcase test app in test fixtures is pure JS. I've been looking at it for a few weeks, I'll convert it to TypeScript at some point :)  So, unit test is our only option here for now.

Reply:
Oops, right, now the unit test is added! Thank you